cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -186,6 +191,76 @@ def is_demultiplexing_possible(self, flow_cell: FlowCellDirectoryData) -> bool:
             demultiplexing_possible = False
         return demultiplexing_possible
 
+    @property
+    def rta_complete_path(self) -> Path:
+        """Return RTAComplete path."""
+        return Path(self.path, DemultiplexingDirsAndFiles.RTACOMPLETE)
+
+    @property
+    def copy_complete_path(self) -> Path:
+        """Return copy complete path."""
+        return Path(self.path, DemultiplexingDirsAndFiles.COPY_COMPLETE)
+
+    @property
+    def demultiplexing_started_path(self) -> Path:
+        """Return demux started path."""
+        return Path(self.path, DemultiplexingDirsAndFiles.DEMUX_STARTED)
+
+    @property
+    def trailblazer_config_path(self) -> Path:
+        """Return file to SLURM job ids path."""
+        return Path(self.path, "slurm_job_ids.yaml")
+
+    @property
+    def is_demultiplexing_complete(self) -> bool:
+        return Path(self.path, DemultiplexingDirsAndFiles.DEMUX_COMPLETE).exists()
+
+    def has_demultiplexing_started_locally(self) -> bool:
+        """Check if demultiplexing has started path exists on the cluster."""
+        return self.demultiplexing_started_path.exists()
+
+    def has_demultiplexing_started_on_sequencer(self) -> bool:
+        """Check if demultiplexing has started on the NovaSeqX machine."""
+        latest_analysis: Path = get_latest_analysis_path(self.path)
+        if not latest_analysis:
+            return False
+        return Path(
+            latest_analysis, DemultiplexingDirsAndFiles.DATA, DemultiplexingDirsAndFiles.BCL_CONVERT
+        ).exists()
+
+    def is_sequencing_done(self) -> bool:
+        """Check if sequencing is done.
+        This is indicated by that the file RTAComplete.txt exists.
+        """
+        LOG.info("Check if sequencing is done")
+        return self.rta_complete_path.exists()
+
+    def is_copy_completed(self) -> bool:
+        """Check if copy of flow cell is done.
+        This is indicated by that the file CopyComplete.txt exists.
+        """
+        LOG.info("Check if copy of data from sequence instrument is ready")
+        return self.copy_complete_path.exists()
+
+
+def is_flow_cell_ready(self) -> bool:
```

Comment:
Not sure where this function belongs, but it probably should accept the path to the flow cell directory